packages/protocol/contracts/governance/Validators.sol

```diff
@@ -755,7 +751,6 @@ contract Validators is
    * @param account The group to fetch slashing multiplier for.
    */
   function getValidatorGroupSlashingMultiplier(address account) external view returns (uint256) {
```

Comment:
DITTO: Can't modify contract to return 1 on L2 without going over `maxCodeSize`. Enabled `resetSlashingMultiplier` on L2 instead.